evaluation/loogle/calculate_metrics.py

```diff
@@ -95,10 +82,10 @@ def wrapped_metric(answer, predicted_answer):
     return wrapped_metric
 
 
-def calculate_metrics(df: pd.DataFrame) -> Dict:
+def calculate_metrics(df: pd.DataFrame) -> dict:
     nltk.download("wordnet")
 
-    scores: Dict = {}
+    scores: dict = {}
```

Comment:
not sure typing required here, ok to leave it

Reply:
IICR `make format` failed without it